UefiCpuPkg/Library/CpuPageTableLib/CpuPageTableMap.c

```diff
@@ -390,6 +390,17 @@ PageTableLibMapInLevel (
       if (RETURN_ERROR (Status)) {
         return Status;
       }
+
+      OneOfPagingEntry.Pnle.Uint64 = 0;
+      // MU_CHANGE Start - Populate base address bits for non present pages
+      // TODO: Author unit test to verify the split pages meet expected attribute bits.

```

Comment:
Is there an issue or something else tracking this todo?

Reply:
Not that I am aware of, I can open one.

Reply:
Removed TODO statement, and opened #1082 to track TODO statement.